style.css

```diff
@@ -0,0 +1,100 @@
+body {
+  background: #191765;
+  font-family: Roboto, sans-serif;
+  color: #fff;
+  padding: 0;
+  margin: 0;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+}
+
+li {
+  font-size: 14px;
+  margin-top: 10px;
+}
+
+.contact ul {
+  display: flex;
+  justify-content: center;
+  margin: 10px 0 0;
+}
+
+.content {
+  text-align: center;
+  background-image: url("image/Ellipse18.svg");
+  background-repeat: no-repeat;
+  background-position: 0 -50px;
+  padding: 25px 0  0 60px;
```

Comment:
Kindly, stick to the Figma design remove the padding, and since you already used `display: flex` and `justify-content: center` to center everything.